services/ses/index.js

```diff
@@ -17,6 +17,11 @@ function filterInternalErrors(error) {
   throw error;
 }
 
+function validateContact(contact) {
+  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/; // eslint-disable-line max-len
```

Comment:
(Beware, `regex + emails` lead to madness. To be fair, you can replace `emails` with pretty much anything and have this still be a true statement.)